Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py

```diff
@@ -586,6 +596,48 @@ def tc_indicator_get_tags(tc, owners, indicator_value, indicator_type):
     return tags
 
 
+def tc_indicator_get_attributes(tc, owners, indicator_value, indicator_type):
+    attributes = []
+    types = tc_get_indicator_types_request()['data']['indicatorType']
+    for item in types:
+        if indicator_type is not None:
+            if item['name'] == indicator_type:
+                apiBranch = item['apiBranch']
+        else:
+            # meaning we got an indicator but SDK returned a null type
+            apiBranch = None
+
+    ro = RequestObject()
+    ro.set_http_method('GET')
+    ro.set_owner(owners)
+    if apiBranch is not None:
+        ro.set_request_uri("/v2/indicators/{}/{}/attributes".format(apiBranch, quote(indicator_value).replace("/", "%2F")))
+        results = tc.api_request(ro)
+        if results.headers['content-type'] == 'application/json':
+            if 'data' in results.json():
+                if 'attribute' in results.json()['data']:
+                    attributes = results.json()['data']['attribute']
+            else:
+                attributes = []
+    else:
+        try:
+            for item in types:
+                ro.set_request_uri(
+                    "/v2/indicators/{}/{}/attributes".format(item['apiBranch'], quote(indicator_value, safe='')))
+                results = tc.api_request(ro)
+                if results.headers['content-type'] == 'application/json':
+                    if 'data' in results.json():
+                        if 'attribute' in results.json()['data']:
+                            attributes = results.json()['data']['attribute']
+                            break
+                    else:
+                        attributes = []
+        except Exception as error:
+            demisto.error(str(error))
+
+    return attributes
+
```

Comment:
Hit the ThreatConnect API via the URI "/v2/indicators/{apiBranch}/{indicator_value}/attributes" and return the array of attributes (dictionary object which contains the keys: "dateAdded", "displayed", "id", "lastModified", "type", and "value"). Type being the actual attribute name and the value is the contents of the attribute for the respective indicator